build.rs

```diff
@@ -40,6 +40,24 @@ fn macos_static_ruby_dep() {
 #[cfg(not(target_os = "windows"))]
 fn windows_static_ruby_dep() {}
 
+fn rustc_cfg_ruby_version() {
+    let program_version = rbconfig("RUBY_PROGRAM_VERSION");
+    let v: Vec<u32> = program_version.split('.').map(|s| s.parse::<u32>().unwrap()).collect();
```

Comment:
This is great. I just want to leave a comment, because RUBY_PROGRAM_VERSION works for preview releases too, so this code won't break in cases where you're using a pre-release or an RC:

```
➜  ~ chruby 2.7.0-preview1
➜  ~ irb
irb(main):001:0> RbConfig::CONFIG["RUBY_PROGRAM_VERSION"]
=> "2.7.0"
irb(main):002:0> 
```